htdocs/web_portal/views/project/view_project.php

```diff
@@ -46,7 +46,7 @@
         <span class="header" style="vertical-align:middle; float: left; padding-top: 0.9em; padding-left: 1em;">
             This project <?php if ($ngiCount == 0){echo "has no";} else{echo "consists of " . $ngiCount;} ?> NGI<?php if($ngiCount != 1) echo "s"?>
         </span>
-        <img src="<?php echo \GocContextPath::getPath()?>img/ngi.png" height="25px" style="float: right; padding-right: 1em; padding-top: 0.5em; padding-bottom: 0.5em;" />
+        <img src="<?php echo \GocContextPath::getPath()?>img/ngi.png" class="decoration" />
```

Comment:
Whilst moving to a CSS class is very positive, is there a reason to only do it for this icon (and not for example the trash icons)?

Reply:
Because a `decoration` CSS class already existed, whereas I don't think a class exists for the edit/delete icons.

Reply:
perhaps an issue